roles/openshift_storage_glusterfs/tasks/heketi_deploy_part2.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Create heketi DB volume
-  command: "{{ glusterfs_heketi_client }} setup-openshift-heketi-storage --listfile /tmp/heketi-storage.json"
+  command: "{{ glusterfs_heketi_client }} setup-openshift-heketi-storage --image {{ glusterfs_heketi_image}}:{{ glusterfs_heketi_version }} --listfile /tmp/heketi-storage.json"
```

Comment:
I beleive the `--image` tag is no longer supported in heketi, as it was just a workaround until it was able to reuse the image specified in the Pod spec. Remove this.